external/buildscripts/build.pl

```diff
@@ -1509,6 +1512,20 @@
 			system("rm -rf $distdirlibmono/gac/nunit*");
 		}
 
+		if ($buildMSBuild)
+		{
+			#Our classlibrary build is missing String.format, so discard our changes before building msbuild
+			my $currentPath = $ENV{PATH};
+			$ENV{PATH} = $startPath;
+
+			chdir("$msbuildroot") eq 1 or die ("failed to chdir : $monoroot/external/msbuild\n");
+			system("./cibuild.sh --scope Compile --target Mono --host Mono --config Release") eq 0 or die ("failed to build msbuild");
+			system("./install-mono-prefix.sh $distdir") eq 0 or die ("failed to install mono");

```

Comment:
We do we want to install MSBuild with everything else, or to it's own install folder? I am worried about polluting out Mono build folder with MSBuild output. Thoughts? For example, I plan on removing Boo/Unityscript from there too.

Reply:
I figured we'd package it the same way they do, but we should be able to copy it anywhere. I think this mostly puts things in builds/monodistribution/lib/mono/msbuild/15.0/bin, but I haven't fully compared the two monodistribution folders yet